app/controllers/helpers.rb

```diff
@@ -0,0 +1,55 @@
+module Helpers
+	def self.available
+		true
+	end
+
+	def self.can_edit?(session, tweet)
+		#binding.pry
+		if session[:id]
+			tweet.user.id==User.find(session[:id]).id
+		else
+			false
+		end
+	end
+
+	def self.modify_tweet(tweet, params)
```

Comment:
You've reimplemented `tweet.update({:content => params[:content])` with this helper. Look at ActiveRecord [`update`](http://api.rubyonrails.org/classes/ActiveRecord/Relation.html#method-i-update) and the other ways of [updating objects with activerecord](http://www.davidverhasselt.com/set-attributes-in-activerecord/).